drivers/xen/xenbus.c

```diff
@@ -471,7 +480,8 @@ int xenbus_read_integer(const char *path)
 	return t;
 }
 
-int xenbus_read_uuid(const char *path, unsigned char uuid[16]) {
+int xenbus_read_uuid(const char *path, unsigned char uuid[16])
```

Comment:
here and elsewhere: the reviewer has requested "Document functions, please."
What is the reason not documenting this one?